uktrade/data-hub-frontend
# Preselect radio buttons on the interactions form

## Description of change
When a user creates an interaction we now save the `theme` and `kind` into [localStorage](https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage). The next time the user creates an interaction those values are retrieved from `localStorage` and used to pre-select the `theme` and `kind` radio buttons saving the user time completing the form. We know from statistics that users select the same `theme` and `kind` 85% of the time.

## Test instructions
1. Go to: `/companies/<company-uuid>/interactions/create?step=interaction_type`
2. Select a `theme` (Export, Investment, Trade agreement or Other)
3. Select a `kind` (interaction to service delivery) only applies to export and other.
4. Click continue
5. Complete the rest of the form

**After the form saves  both the `them` and `kind` are written to `localStorage` which can be viewed in Chrome DevTools.**

6. Go to: `/companies/<company-uuid>/interactions/create?step=interaction_type` and observe the radio button selection based on the previous selection.

## Screenshots

### Before

_Add a screenshot_

### After

_Add a screenshot_

## Checklist

[//]: # 'When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/main/docs/Code%20review%20guidelines.md'
